src/helpers/tags.ts

```diff
@@ -17,6 +17,13 @@ export const CI_BUILD_LEVEL = '_dd.ci.build_level'
 export const GIT_BRANCH = 'git.branch'
 export const GIT_SHA = 'git.commit.sha'
 export const GIT_TAG = 'git.tag'
+export const GIT_COMMIT_MESSAGE = 'git.commit.message'
+export const GIT_COMMIT_COMMITTER_DATE = 'git.commit.committer.date'
+export const GIT_COMMIT_COMMITTER_EMAIL = 'git.commit.committer.email'
+export const GIT_COMMIT_COMMITTER_NAME = 'git.commit.committer.name'
+export const GIT_COMMIT_AUTHOR_DATE = 'git.commit.author.date'
+export const GIT_COMMIT_AUTHOR_EMAIL = 'git.commit.author.email'
+export const GIT_COMMIT_AUTHOR_NAME = 'git.commit.author.name'
```

Comment:
Nit: could we sort alphabetically these constants?

Reply:
sure thing